src/NuGet.Core/NuGet.Packaging/Signing/Archive/ExtraField.cs

```diff
@@ -37,7 +37,6 @@ internal static bool TryRead(CentralDirectoryHeader header, out IReadOnlyList<Ex
 
             return TryRead(
                 header.ExtraField,
-                header.ExtraFieldLength,
```

Comment:
This is part of zip file spec, maybe we shouldn't remove it. https://pkware.cachefly.net/webdocs/casestudies/APPNOTE.TXT

According to wiki it's where some OS specific or zip archiver specific meta data is saved, could be useful in the future.

Reply:
If it's never used why does it matter?